Add Galactic feature note for externally configuring QoS (#1033)
ros2/ros2_documentation: source/Releases/Release-Galactic-Geochelone.rst

Related PRs: `ros2/rcl_logging#53 <https://github.com/ros2/rcl_logging/pull/53>`_ and `ros2/launch#460 <https://github.com/ros2/launch/pull/460>`_.

Externally configure QoS at start-up
------------------------------------

It is now possible to externally configure the QoS settings for a node at start-up time.

> QoS settings are **not** configurable during runtime and node authors must opt-in to enable this feature.

Reviewer: this seems a bit confusing to me, maybe:

```suggestion
QoS settings are configurable during runtime only if the node authors have enabled this feature.
```

Reviewer 2: @ivanpauno I think that rewrite completely changes the meaning, and suggests that QoS *can* be changed via runtime.  So I don't think we want to say that.

Really, I think this sentence should be split into two because it is talk about two separate topics.  Something like:

```
QoS settings are **not** configurable during runtime; they are only configurable at start-up time.
In order to change the QoS settings at runtime, node authors must opt-in to enable this feature.
```

Or something like that.  The previous sentence may need to be tweaked as well.

Reviewer: ah ok, I got it now.
The distinction between "runtime" and "startup time" can be confusing (because "startup time" is also "runtime").

> Really, I think this sentence should be split into two because it is talk about two separate topics

:+1: your suggested wording looks fine!

Author: I've updated the wording in 792ccbd
